controllers/feeds/getAnonymousFeedChat.js

```diff
@@ -36,10 +36,11 @@ const getAnonymousFeedChatService = async (req, res) => {
         constantActor,
         isAnonym,
         isOwnSignedPost,
+        isOwnAnonymousPost,
         isOwnPost,
         message,
         actor
-      } = getDetail(req, b, mySignedId);
+      } = getDetail(req, b, mySignedId, req?.userId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getDetail` function call has been updated with new arguments. However, it's not clear if the function definition has been updated to handle these new arguments. Please ensure that the function definition of `getDetail` is updated to handle `mySignedId` and `req?.userId`.

```diff
- } = getDetail(req, b, mySignedId);
+ } = getDetail(req, b, mySignedId, req?.userId);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->